app/CMakeLists.txt

```diff
@@ -1,57 +1,62 @@
-add_executable( exiv2
-     exiv2.cpp
-     exiv2app.hpp
-     actions.cpp actions.hpp
-     getopt.cpp getopt.hpp
-     app_utils.cpp app_utils.hpp
+add_executable(
+  exiv2
```

Comment:
Is there a way to keep the target separate from the sources list, i.e. still on the first line, or have sources indented more?

Reply:
The way to do it is to have a separate variable. Did that.